packages/kit/src/views/Discovery/pages/Dashboard/Welcome/Welcome.tsx

```
@@ -1,27 +1,104 @@
+
+import { useMemo } from 'react';
```

ESLint

[error] Delete `⏎`

(prettier/prettier)